ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -28,9 +29,10 @@ import (
 )
 
 const (
-	RayJobDefaultRequeueDuration    = 3 * time.Second
-	RayJobDefaultClusterSelectorKey = "ray.io/cluster"
-	PythonUnbufferedEnvVarName      = "PYTHONUNBUFFERED"
+	RayJobDefaultRequeueDuration        = 3 * time.Second
+	RayJobDefaultClusterSelectorKey     = "ray.io/cluster"
+	PythonUnbufferedEnvVarName          = "PYTHONUNBUFFERED"
+	DELETE_RAYJOB_CR_AFTER_JOB_FINISHES = "DELETE_RAYJOB_CR_AFTER_JOB_FINISHES"
```

Comment:
can you move this env var to `constant.go`?

Reply:
Done